src/Apps/Legion/Components/Diagram/Internal/Annotation/Annotation.styles.ts

```diff
@@ -0,0 +1,20 @@
+import { FontWeights } from '@fluentui/react';
+import { TAnnotationClassNames } from './Annotation.types';
+import { makeStyles } from '@fluentui/react-components';
+import { shorthands } from '@griffel/react';
+
+export const useClassNames = makeStyles<TAnnotationClassNames>({
+    root: {
+        width: 'fit-content',
+        color: 'black',
```

Comment:
is the hard coding of black intentional here since we are assuming the image is going to have a white background? Could we make the annotations chagne with the theme and still have things show up properly?

Reply:
yeah since the images we have always have white background, that is why I kept it there as black. But I changed it to theme.palette.blueDark to change it with theme. It was the closest to black which shows always a dark color no matter what the selected theme is.